Platform/TigerlakeBoardPkg/Library/Stage1ABoardInitLib/Stage1ABoardInitLib.c

```diff
@@ -46,7 +46,7 @@ FSPT_UPD TempRamInitParams = {
     .PcdSerialIoUartMode        = 4, // SerialIoUartSkipInit, let SBL init UART
     .PcdSerialIoUartBaudRate    = 115200,
     .PcdPciExpressBaseAddress   = FixedPcdGet32 (PcdPciMmcfgBase),
-    .PcdPciExpressRegionLength  = 0x20000000,
+    .PcdPciExpressRegionLength  = 0x10000000,

```

Comment:
do we need to sync up  FSP-T UPD PcdPciExpressRegionLength in 
Platform\TigerlakeBoardPkg\Library\Stage1ABoardInitLib\Stage1ABoardInitLib.c ?

Reply:
256 buses × 32 devices × 8 functions × 4 KB = 256 MB =0x10000000, so we use 0x10000000 for MR2 release.
I am not sure why 0x20000000 was used before, but the new value make more sense.